karma.conf.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function (config) {
+	config.set({
+		basepath: '',
+		frameworks: ['jasmine'],
```

Comment:
Let use mocha, instead of jasmine. I think mocha is widely used. The only change needed to do this is changing `test` to `it` or `describe`.

Reply:
What assertion library should you suggest when using `mocha`, should I go with `chai`?

Reply:
I don't think we will need a assertion library. We can just user the node.js `assert` module to do the assertions.